node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py

```diff
@@ -19,10 +19,30 @@ def test_node_declaration_signed_change_request_can_be_sent_via_api(api_client,
     assert signed_change_request.signature
 
     payload = signed_change_request.dict()
-    # TODO(dmu) CRITICAL: Expect `assert response.status_code == 204` instead once `perform_create()` is implemented
-    #                     https://thenewboston.atlassian.net/browse/BC-167
-    with pytest.raises(NotImplementedError, match=f'"identifier": "{regular_node.identifier}"'):
-        api_client.post('/api/signed-change-request/', payload)
+    response = api_client.post('/api/signed-change-request/', payload)
+    assert response.status_code == 201
```

Comment:
I think that if the signed change request is accepted by PV and it is added to the blockchain before the response is sent (inside the request handler code) then HTTP201 must be returned, in case we just accept the request and put it to a queue (we will implement it later), then HTTP204 must be returned. CV and regular node responds with the same code they got from PV

Reply:
I added stubs to the right places. Also I created new Jira task [https://thenewboston.atlassian.net/browse/BC-190](https://thenewboston.atlassian.net/browse/BC-190)